gap/oper.gi

```diff
@@ -608,15 +608,19 @@ end);
 InstallMethod(OnDigraphs, "for a mutable digraph by out-neighbours and a perm",
 [IsMutableDigraph and IsDigraphByOutNeighboursRep, IsPerm],
 function(D, p)
-  local out;
-  if ForAll(DigraphVertices(D), i -> i ^ p = i) then
+  local out, permed;
+  if p = () then
```

Comment:
In one sense this is worse: previously we were testing whether `p` acts as the identity on `DigraphVertices(D)`, we can be the case even if `p` is not the identity permutation. But I don't mind too much about losing that case.

Is `p = ()` optimised somehow? Do all permutations inherently know whether or not they are the identity, for instance?

Reply:
P=() isn't specifically optimised, but it will almost always be very fast, espically if (as is common) the permutation moves the largest point it is defined on, as that is checked first.

Reply:
Just to say, we could use `SmallestMovedPoint` is we really care about saving this optimisation. That is still much faster than the old code, and would keep the same answer.